addons/account_peppol/wizard/account_move_send.py

```diff
@@ -71,6 +72,19 @@ def _compute_enable_peppol(self):
                 )
             )
 
+    @api.depends('company_id.account_edi_proxy_client_ids.edi_mode')
+    def _compute_account_peppol_edi_mode_info(self):
+        mode_strings = {
+            'test': _('Test'),
+            'demo': _('Demo'),
+        }
+        for wizard in self:
+            edi_user = wizard.company_id.account_edi_proxy_client_ids.filtered(
+                lambda usr: usr.proxy_type == 'peppol'
+            )
+            mode = mode_strings.get(edi_user.edi_mode)
+            wizard.account_peppol_edi_mode_info = mode and ' (%s)' % mode or ''
```

Comment:
```suggestion
            wizard.account_peppol_edi_mode_info = mode and f' ({mode})' or ''
```
Cosmetic remark: It's more readable to use if/else than and or
```suggestion
            wizard.account_peppol_edi_mode_info = f' ({mode})' if mode else ''
```